spec/controllers/idv/capture_doc_status_controller_spec.rb

```diff
@@ -0,0 +1,147 @@
+require 'rails_helper'
+
+describe Idv::CaptureDocStatusController do
+  let(:user) { build(:user) }
+  let(:document_capture_step_enabled) { false }
+
+  before do |example|
+    stub_sign_in(user) unless example.metadata[:skip_sign_in]
```

Comment:
since only one test has the `skip_sign_in` metadata, WDYT of just grouping the contexts/before filters differently (or using a `let`)? I feel like we tend to not use example metadata because it's easy to miss

Reply:
> since only one test has the `skip_sign_in` metadata, WDYT of just grouping the contexts/before filters differently (or using a `let`)? I feel like we tend to not use example metadata because it's easy to miss

Good call. Originally I think I liked the idea of example metadata (and [based on some other test specs](https://github.com/18F/identity-idp/blob/538d3415b40fe7f867dc37852dcc910c2f3e9f02/spec/controllers/idv/doc_auth_controller_spec.rb#L16)) as a way to avoid needing to assign some shared user value, though that was before I'd implemented `let(:user)` anyways. With that in mind, it seems easiest to just use that value, and override it in the "unauthenticated" context.

Updated in b573ed5.